app/controllers/homepages_controller.rb

```diff
@@ -0,0 +1,16 @@
+class HomepagesController < ApplicationController
+  def index
+    # @works = Work.all
+    # @albums = Work.where(category: "album")
+    # @books = Work.where(category: "book")
+    # @movies = Work.where(category: "movie")
+
+    # Jordan suggested I can merge these into a hash. This can DRYed up my code in views.
```

Comment:
Clever implementation :) Hurray for hashes. You could also consider how you could use a partial view to dry up the index view.